db/scripts/fix_event_headings_with_empty_strings.rb

```diff
@@ -0,0 +1,23 @@
+# We introduced `Events::TransitionDescription` as a consistent entrypoint
+# for describing transitions between events.
+# This brings how we describe Event headings in line with how we describe lists of
+# event modifications.
+# We need to migrate existing data, though.
+
+# Retrieve all the events with headings that "changed from ''"
+#
+# Before: "Induction status changed from '' to 'InProgress'"
+# After: "Induction status set to 'InProgress'"
+Event.where("heading LIKE ?", "%changed from ''%").find_each do |event|
+  heading = event.heading.gsub("changed from ''", "set")
+  event.update_column(:heading, heading)
+end
+
+# Retrieve all the events with headings that "changed from `Something` to ''"
+#
+# Before: "Induction status changed from 'InProgress' to ''"
+# After: "Induction status 'InProgress' removed"
+Event.where("heading LIKE ?", "%to ''").find_each do |event|
+  heading = event.heading.gsub("changed from ", "").gsub("to ''", "removed")
+  event.update_column(:heading, heading)
+end
```

Comment:
I'm not sure we need to do this part.

In Production there are 719 events `WHERE heading LIKE '%changed from ''''%';` and 719 events `WHERE heading LIKE '%''''%';`.